modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/DartClientCodegen.java

```diff
@@ -474,34 +465,62 @@ public String toDefaultValue(Schema schema) {
 
     @Override
     public String getTypeDeclaration(Schema p) {
```

Comment:
can you add unit tests to this method?

Reply:
There are tests for dart and dart-dio that test the type returned for `array` and `map` OAS types.